nucleus/modelci/unit_test.py

```diff
@@ -5,52 +5,21 @@
 and have confidence that they’re always shipping the best model.
 """
 from dataclasses import dataclass, field
-from enum import Enum
 from typing import List
 
-import requests
-
-from nucleus.connection import Connection
-from nucleus.constants import NAME_KEY, SLICE_ID_KEY
-
-from .constants import (
-    EVAL_FUNCTION_ID_KEY,
-    ID_KEY,
-    THRESHOLD_COMPARISON_KEY,
-    THRESHOLD_KEY,
-    UNIT_TEST_ID_KEY,
-    UNIT_TEST_NAME_KEY,
-)
+from ..connection import Connection
+from ..constants import NAME_KEY, SLICE_ID_KEY
+from .data_transfer_objects.eval_function import EvaluationCriteria
+from .data_transfer_objects.unit_test_evaluations import GetEvalHistory
+from .data_transfer_objects.unit_test_metric import AddUnitTestMetric
 from .unit_test_evaluation import UnitTestEvaluation
+from .unit_test_metric import UnitTestMetric
 
-EVALUATIONS_KEY = "evaluations"
-
-
-class ThresholdComparison(Enum):
-    """Comparator between the result and the threshold."""
-
-    GREATER_THAN = "greater_than"
-    GREATER_THAN_EQUAL_TO = "greater_than_equal_to"
-    LESS_THAN = "less_than"
-    LESS_THAN_EQUAL_TO = "less_than_equal_to"
-
-
-@dataclass(frozen=True)
-class UnitTestMetric:
-    """A Unit Test Metric is an evaluation function and comparator associated with a Unit Test.
-    Note that this class is immutable.
 
-    Attributes:
-        unit_test_id (str): The ID of the associated unit test.
-        eval_function_id (str): The ID of the associated evaluation function.
-        threshold (float): The threshold for the unit test.
-        threshold_comparison (ThresholdComparison): The comparator for the unit test.
-    """
-
-    unit_test_id: str
-    eval_function_id: str
-    threshold: float
-    threshold_comparison: ThresholdComparison
+@dataclass
+class UnitTestInfo:
+    name: str
+    slice_id: str
```

Comment:
I actually got rid of this class because the user never uses this, and it felt a little silly to just have a class of this form for no real purpose. I'm wondering if we can just get rid of the `info()` method because it's not super useful, and users will never call it either.

Reply:
Removing!